specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -86,6 +86,12 @@ For every new component on the diagram, explain which goals does it solve.
 In this context, a component is any separate software process.
 
 -->
+#### Package publishing
+The shared-component and frontend packages are published to NPMJS(https://www.npmjs.com/) under the user versatiledatakit.
```

Comment:
Why NPMJS? Why not GitHub Package Registry for example. 
 
For example we decided to use Gitlab Helm registry because their free option had more storage available that we might need . 


Make sure to update the variables definitions in https://github.com/vmware/versatile-data-kit/wiki/Gitlab-CICD#cicd-demo-installation-variables + Setting the actual values in Lastpass and Gitlab CI (when appropriate). Make sure passwords are "protected" in Gitlab CI Variables.

Reply:
Variables definitions updated. 
Variables are already set in lastpass and gitlab. 
We chose NPMJS because it is much more widely adopted @ivakoleva  do you have more to add here?

Reply:
description updated with reasoning.